dotcom-rendering/src/web/components/MainMedia.tsx

```diff
@@ -49,13 +49,26 @@ const immersiveWrapper = css`
     overflow: hidden;
 `;
 
-const chooseWrapper = (format: ArticleFormat) => {
+const chooseWrapper = (format: ArticleFormat, isMainMediaTest?: boolean) => {
 	switch (format.display) {
 		case ArticleDisplay.Immersive:
 			return immersiveWrapper;
 		case ArticleDisplay.Standard: {
 			switch (format.design) {
 				case ArticleDesign.LiveBlog:
+					if (isMainMediaTest)
+						return css`
+							${until.desktop} {
+								margin-left: -20px;
+								margin-right: -20px;
+							}
+
+							${until.mobileLandscape} {
+								margin-left: -10px;
+								margin-right: -10px;
+							}
```

Comment:
It feels strange that this is not the same as `noGutters`. This variation on breakpoint seems as though it might be a very small design tweak which is exploding complexity